dotnet/sample-apps/aws-sdk/wrapper/SampleApps/build.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+
+mkdir -p build/dotnet
+git clone https://github.com/open-telemetry/opentelemetry-dotnet-contrib.git
```

Comment:
This script looks like it's not deterministic since this doesn't reference a git hash and not rerunnable since I don't think git clone works if the folder is already there. How about we add this as a git submodule?

Reply:
Updated to git submodule. Also in the `aws-otel-lambda` repo, we'll have to update the [option](https://github.com/aws-observability/aws-otel-lambda/blob/main/.github/workflows/main-build.yml#L48) from `true` to `recursive` to recursively checkout submodules from each folder.